src/libpriv/rpmostree-bwrap.c

```diff
@@ -322,10 +324,22 @@ rpmostree_bwrap_run (RpmOstreeBwrap *bwrap,
     const char *errmsg = glnx_strjoina ("Executing bwrap(", bwrap->child_argv0, ")");
     GLNX_AUTO_PREFIX_ERROR (errmsg, error);
 
-    if (!g_spawn_sync (NULL, (char**)bwrap->argv->pdata, (char**) bwrap_env, G_SPAWN_SEARCH_PATH,
-                       bwrap_child_setup, bwrap,
-                       NULL, NULL, &estatus, error))
+    g_autofree char *cmd_stdout = NULL;
+    g_autofree char *cmd_stderr = NULL;
+    if (!g_spawn_sync (NULL, (char**)bwrap->argv->pdata, (char**) bwrap_env,
+                       G_SPAWN_SEARCH_PATH, bwrap_child_setup, bwrap,
+                       /* only capture output if requested, otherwise let inherit */
+                       out_cmd_stdout ? &cmd_stdout : NULL,
```

Comment:
This loses streaming (and rate limiting imposed by the journal, etc.)  I was more thinking we'd use `sd_journal_stream_fd()`.  I believe that's basically what the systemd core does when it goes to execute a new process.